src/utils/events.js

```diff
@@ -21,6 +21,7 @@ export function formatEventsForCalendar(events) {
       start: evt.startDate,
       end: evt.endDate,
       title: evt.eventName,
+      type: evt.eventType,
```

Comment:
Added this prop so it could be read on the calendar view (and then change the calendar view's event color).